src/routes/safe/components/Settings/ManageOwners/OwnerAddressTableCell/index.jsx

```diff
@@ -3,17 +3,23 @@ import * as React from 'react'
 import Block from '~/components/layout/Block'
 import Paragraph from '~/components/layout/Paragraph'
 import Identicon from '~/components/Identicon'
+import EtherScanLink from '~/components/EtherscanLink'
 
 type Props = {
   address: string,
+  showLinks?: boolean,
 }
 
 const OwnerAddressTableCell = (props: Props) => {
-  const { address } = props
+  const { address, showLinks } = props
   return (
     <Block justify="left">
       <Identicon address={address} diameter={32} />
-      <Paragraph style={{ marginLeft: 10 }}>{address}</Paragraph>
+      { showLinks ? (
+        <div style={{ marginLeft: 10}}>
```

Comment:
> {{ marginLeft: 10}}

Missing space. Please don't forget to use Prettier & ESLint

Reply:
Done